Duplicati/Library/Utility/WorkerThread.cs

```diff
@@ -254,7 +254,7 @@ public Tx CurrentTask
         {
             get
             {
-                return m_currentTask;
+                return m_currentTask ?? throw new InvalidOperationException("m_currentTask is null");

```

Comment:
The return value can be null. As far as I can read the code, it starts out being null.